### PR TITLE
Update release notes for 1.3 to include CON-1476 and CON-1099

### PR DESCRIPTION
**Pull request by ckallianpur**
Signed-off-by: chaitra <chaitra.kallianpur@hpe.com>

### CODE REVIEW

File: release-notes/v1.3.0.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ The following table lists the known issues for HPE CSI Driver for Kubernetes v1.
...
+|CON-1099|csi.k8s|Editing PVC with alllowed mutations but missing overrides fails| Editing a PVC fails with incorrect error if a field is not allowed in "Overrides"  but allowed in "mutation" and present in PVC definition.|Mutation fields need to be present for overrides in storage class to prevent this error.
+|CON-1476|csi.k8s|PVCs parameters which are allowed in mutations and not in overrides are updated automatically|PVC with annotation for parameters which are allowed in "mutation" and not in "Overrides" gets updated automatically after about 2 minutes of time. Volumes get removed from volume collection as well.|Mutation fields need to be present for overrides in storage class to prevent this error.
+
```
Comment (rgcostea) [inline release-notes/v1.3.0.md:25]: These don't seem to be showing up as 2 separate rows in the table.  Are you missing a | somewhere?

Comment (ckallianpur) [reply release-notes/v1.3.0.md:25]: Looks like all the "|" are present. Also in https://github.com/ckallianpur/csi-driver/blob/doc/release-notes/v1.3.0.md it shows it as separate rows.

Comment (rgcostea) [reply release-notes/v1.3.0.md:25]: Click the "Display the rich diff" button at the top of this page.  Do you see 2 rows?

Comment (ckallianpur) [reply release-notes/v1.3.0.md]: updated

**Review by raunakkumar — APPROVED** [review]

**Pull request merged**